tests/test_models.py

```
@@ -2,772 +2,228 @@
+
+@pytest.mark.parametrize(
+    "field,expected_value,expected_type",
+    [
+        ("meta.rc", "ok", str),
+        ("data", 1, len),  # Check length of data list
+        ("data[0].name", "default", str),
+        ("data[0]", Site, Site),  # Check type of first data item
+    ],
+)
+def test_base_response_fields(
+    field: str, expected_value: Any, expected_type: Any
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `expected_value`

(ANN401)